metadetect/defaults.py

```diff
@@ -13,3 +14,34 @@
     384,512,768,1024,1536,  # noqa
     2048,3072,4096,6144  # noqa
 ]
+
+# stamp size is not a default, depends on measurement
+DEFAULT_MDET_CONFIG = {
+    # wmom or ksigma
+    'meas_type': 'wmom',
+
+    # fitgauss or gauss
+    'metacal_psf': 'fitgauss',
+
+    # In units of sky noise.  Default for lsst is 5
+    'detect_thresh': 5.0,
+
+    # deblending always occurs; this says to do the shear measurements on
+    # deblended stamps
+    'use_deblended_stamps': False,
+
+    # do sky sub on each coadd input
+    'subtract_sky': False,
+}
```

Comment:
I don't think this change will serve us well in the long run. Having the options here line in a nested way the functions in `ngmix` (and the same names) makes experimentation, additions, and discovery much easier.

Reply:
old style is back